Portfolio2.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <title>Elmi Farah's Portfolio'</title>
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <link rel="stylesheet"type="text/css"href="portfolio2.css">
+   </head>
+   <body>
+      <header>
+               <ul>
+                  <li class="logo"> <img src="maydenacademylogo.png"/> </li>
```

Comment:
All img tags must have an alt attribute that describes what the image is. This is for users that have visual impairments and use a screen reader (which is actually more than you might think)

Like this example:

`<img src="mayden.jpg" alt="Mayden Logo" />`